src/app/rundown-view/components/minishelf/mini-shelf.component.html

```diff
@@ -14,6 +14,5 @@
         <span class="c-mini-shelf__title">{{ getSanitizedTitle() | uppercase }}</span>
         <span class="c-mini-shelf__duration">{{ mediaDurationInMsWithoutPostroll | timer : 'HH?:mm:ss' }}</span>
     </div>
+    <sofie-mini-shelf-tooltip *ngIf="segment && segment.metadata && segment.metadata.miniShelfVideoClipFile" [fileName]="segment.metadata.miniShelfVideoClipFile"></sofie-mini-shelf-tooltip>
```

Comment:
Why do we even need an `*ngIf` on this? If we have a MiniShelf, we should always show the tooltip for it.

Reply:
You can just use a getter method, or something like it, from the `mini-shelf.component.ts` file if it is to avoid providing undefined in `[fileName]`.

Reply:
Done

Reply:
@vivanov90 There is mention of three different things in this comment thread. It would be nice that in cases like these that you referred to _what_ was done :)

Reply:
@LindvedKrvang this is move in getter:
  ```
public get mediaFilename(): string {
    return this.segment.metadata?.miniShelfVideoClipFile ?? ''
  }
```
`[filename]="mediaFilename"`